src/features/multichain/utils/utils.ts

```diff
@@ -121,11 +121,7 @@ export const predictAddressBasedOnReplayData = async (safeCreationData: Replayed
 }
 
 export const hasMultiChainCreationFeatures = (chain: ChainInfo): boolean => {
-  return (
-    hasFeature(chain, FEATURES.MULTI_CHAIN_SAFE_CREATION) &&
-    hasFeature(chain, FEATURES.COUNTERFACTUAL) &&
-    hasFeature(chain, FEATURES.SAFE_141)
-  )
+  return hasFeature(chain, FEATURES.MULTI_CHAIN_SAFE_CREATION) && hasFeature(chain, FEATURES.COUNTERFACTUAL)
```

Comment:
As multichain required >1.4.1, what do you think about including a semver comparison of `recommendedMasterCopyVersion` too?

Reply:
Why do we not include it here and [only `hasMultiChainAddNetworkFeature`](https://github.com/safe-global/safe-wallet-web/pull/4642/commits/9421c23a41cb8dec1cf2fd322ed767909a7c302c#diff-e224730418c4d512f934ecdd399575e7b8786239ada01bcfc9a7632ee7c8d3a4R133)? Without the comparison here, [it will affect the `NetworkMultiSelector`](https://github.com/safe-global/safe-wallet-web/blob/5c1307c082447e93abb9da1bc59bbe1d7c85656e/src/components/common/NetworkSelector/NetworkMultiSelector.tsx#L72). I think our tests would've picked this up if we weren't [only mocking 1.4.1](https://github.com/safe-global/safe-wallet-web/blob/9421c23a41cb8dec1cf2fd322ed767909a7c302c/src/components/common/NetworkSelector/__tests__/NetworkMultiSelector.test.tsx).